packages/ag-charts-locale/src/en-US.ts

```diff
@@ -81,8 +81,12 @@ export const AG_CHARTS_LOCALE_EN_US: Record<string, string> = {
     toolbarAnnotationsDisjointChannel: 'Disjoint channel',
     // Text for the annotation toolbar's clear all button
     toolbarAnnotationsClearAll: 'Clear all annotations',
-    // Text for the annotation toolbar's color picker annotation button
+    // @deprecated Text for the annotation toolbar's color picker annotation button
```

Comment:
Nit: Add the version deprecated here so it's easy for us to tell when to cleanup? Also there should be clear instruction on what the user should do instead?

e.g.
```
/**
 * Text for the annotation toolbar's color picker annotation button
 * @deprecated v10.1.0 use `toolbarAnnotationsLineColor` and `toolbarAnnotationsTextColor` instead. 
 */
 ```